### PR TITLE
[CFD-86] Outlined Button Component

### PR DESCRIPTION
**Pull request by 1zhaohel**
### Description

Created two of the three outlined buttons. The third outlined button will have to take in an icon so waiting for a PR. 

### Reason for Change

Completes CFD-86

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Testing

Tested the two buttons in `index.tsx`
image
image
image

```
<View className="flex flex-1 flex-col items-center justify-start">
  <View className="w-30">
    <OutlinedButton onPress={sayHello} fill={false}>
      Label
    </OutlinedButton>
  </View>
</View>
```

### CODE REVIEW

File: apps/expo/src/components/OutlinedButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Text, TouchableOpacity, View } from "react-native";
+
+interface OutlinedButtonProps {
+  onClick: () => void;
```
Comment (BakerWJ) [inline apps/expo/src/components/OutlinedButtons.tsx:5]: change this to onPress for consistency

**Review by BakerWJ — COMMENTED** [review]
the text in the image doesn't look centered in the screenshot, so may want to confim that it is centered

**Comment by 1zhaohel** [comment]
> the text in the image doesn't look centered in the screenshot, so may want to confim that it is centered

i see it too - to change i made the vertical padding 8px instead of 10px (from figma) since 8 looks more centered?

File: apps/expo/src/components/OutlinedButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const OutlinedButton: React.FC<OutlinedButtonProps> = ({
+  onPress,
+  children,
+  fill,
+}) => {
+  return (
+    <TouchableOpacity onPress={onPress}>
+      <View className="h-12 flex-auto items-start gap-2">
+        <View
+          className={`border-p-40 rounded-full border bg-white px-6 py-2 ${
```
Comment (BakerWJ) [inline apps/expo/src/components/OutlinedButtons.tsx:20]: is there a better way to center the text than by using padding?

Comment (1zhaohel) [reply apps/expo/src/components/OutlinedButtons.tsx:20]: I tried a lot of methods like `items-center`, `content-center`, messing with gap and width, structuring the views differently, but couldn't find any other way without the button losing its structure TT. Do you know any way to solve this issue?

Comment (BakerWJ) [reply apps/expo/src/components/OutlinedButtons.tsx:20]: ok thats fine, as long as you tried other options

**Review by BakerWJ — APPROVED** [review]

File: apps/expo/src/components/OutlinedButtons.tsx (hunk 2 above)
Comment (BakerWJ) [inline apps/expo/src/components/OutlinedButtons.tsx:20]: remove nested views

**Review by BakerWJ — CHANGES_REQUESTED** [review]
take a look at the changes in #355 -- these nested views seem to be redundant for the button

File: apps/expo/src/components/OutlinedButtons.tsx (hunk 2 above)
Comment (1zhaohel) [reply apps/expo/src/components/OutlinedButtons.tsx:20]: okay sounds good! should i also change how FilledButtons is structured?

Comment (BakerWJ) [reply apps/expo/src/components/OutlinedButtons.tsx:20]: Ryan already made changes to those ones, so you can just modify the Outlined one

File: apps/expo/src/components/OutlinedButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Text, TouchableOpacity } from "react-native";
+
+interface OutlinedButtonProps {
+  onPress: () => void;
+  children: React.ReactNode;
+  fill: boolean;
+  // icon to be added
```
Comment (BakerWJ) [inline apps/expo/src/components/OutlinedButtons.tsx:8]: why does the icon need to be added? it should be able to be inserted as a child like `<Button>Baker <Icon /> </Button`

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**